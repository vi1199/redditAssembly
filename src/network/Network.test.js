import React from 'react';
import renderer, {act} from 'react-test-renderer';
import HomeScreen from '../home/screens/Home';

jest.mock('./picsMockData', () => jest.fn());
describe('App component', () => {
  it('Home screen loads with data', async () => {
    let tree;
    await act(async () => {
      tree = renderer.create(<HomeScreen />);
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
