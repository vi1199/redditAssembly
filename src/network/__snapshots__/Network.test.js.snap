// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component it works 1`] = `
<RCTSafeAreaView
  accessibilityLabel="view"
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#D0D0D0",
      "flex": 1,
    }
  }
>
  <TextInput
    onChangeText={[Function]}
    placeholder="Search..."
    style={
      Object {
        "backgroundColor": "#E0E0E0",
        "height": 40,
        "margin": 10,
        "paddingLeft": 20,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <RCTScrollView
    data={Array []}
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsVerticalScrollIndicator={false}
    stickyHeaderIndices={Array []}
    viewabilityConfigCallbackPairs={Array []}
  >
    <View />
  </RCTScrollView>
</RCTSafeAreaView>
`;
